packages/@aws-cdk/core/lib/assets.ts

```diff
@@ -319,4 +319,10 @@ export interface DockerImageAssetLocation {
    * The name of the ECR repository.
    */
   readonly repositoryName: string;
+
+  /**
+   * The tag of the image in Amazon ECR.
+   * @default ${dockerTagPrefix}${asset.sourceHash}
```

Comment:
```suggestion
   * @default `${dockerTagPrefix}${asset.sourceHash}`
```

does this work? It seems weird to not have the backticks on the default

Reply:
I did not want the docs to actually parse the values, just show that it is the two concatenated. There may be a better way to represent that.

Reply:
This needs to be documentation that makes sense for other languages. This convention is specific to TypeScript/JavaScript and may not translate properly. This should be descriptive of what the default will be, not a code snippet.

Reply:
I changed it to 
`@default - dockerTagPrefix + asset.sourceHash`

I can be more descriptive, say "dockerTagPrefix concatenated with the sourceHash of the asset", but I think the + conveys string concatenation agnostically